### DISCUSSION TITLE
Allow for screen lock option

### DISCUSSION THREAD

**Discussion by yodaluca23** [PoomSmart/YTABConfig#87] [General]
This allows the option for screen lock mode to be enabled, I've only gotten it to show when in landscape however.

Enable:
`YTColdConfig.enableIosLockMode
YTColdConfig.enableLockModeMenuItemInPortraitOnClient
YTHotConfig.iosPlayerClientSharedConfigEnableLockScreenPlaybackProgressFix`

**Comment by CySxL** [comment]
What's the lock screen mode?

Tried enabling all of them on YT 18.44.3 but seems no difference.

**Comment by yodaluca23** [comment]
> What's the lock screen mode?
> 
> Tried enabling all of them on YT 18.44.3 but seems no difference.

It's the option that lets you lock the screen...

**Reply by CySxL** [reply]
you mean orientation? or i don't know how can't you lock the screen if you don't have the tags enabled?

**Reply by yodaluca23** [reply]
Orientation, and accidental presses...![image](https://github.com/PoomSmart/YTABConfig/assets/67206487/1e60aa3d-fb51-4a3c-8d0e-30c8714ac87d)

**Reply by CySxL** [reply]
yeah we called that orientation lock...... otherwise it means physical locking the device/screen......
